relnotes/taskpool.feature.md

```diff
@@ -0,0 +1,15 @@
+### Change of `TaskPool` and `ThrottledTaskPool` internals
+
+Also affects `StreamProcessor` as it uses `ThrottledTaskPool` internally.
+
+Both `TaskPool` and `ThrottledTaskPool` stopped defining internal private
+`OwnedTask` to wrap user-supplied type in. Instead, regular termination hook
+infrastructure is used.
+
+Both also don't try to catch and rethrow exceptions arising from a task - that
+is already done by termination hook system.
```

Comment:
This sounds like catching the exceptions is done by the termination hook system.
Do I understand right, that what you actually mean is: try to catch the exception to call `this.recycle(task)` is not needed because the recycle call is done by the (new) termination hook?

Reply:
Yup

Reply:
Reworded